registration-processor/pre-processor/registration-processor-osi-validator-stage/src/main/java/io/mosip/registration/processor/stages/osivalidator/OSIValidator.java

```diff
@@ -646,6 +658,37 @@ private boolean validateUserBiometric(String registrationId, String userId, List
 		}
 		
 	}
+	
+	/**
+	 *  get the vid by userid
+	 * @param userid
+	 * @return vid 
+	 * @throws ApisResourceAccessException
+	 * @throws IOException
+	 */
+	private String getVidByUserId(String userid) throws ApisResourceAccessException, IOException {
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(), "",
+				"OSIValidator::getVidByUserId():: entry");
+		List<String> pathSegments = new ArrayList<>();
+		pathSegments.add(APPID);
+		pathSegments.add(userid);
+		String vid = null;
+		ResponseWrapper<?> response = null;
+		regProcLogger.debug(LoggerFileConstant.SESSIONID.toString(), LoggerFileConstant.USERID.toString(), "",
+				"OSIValidator::getUinByVid():: GETVIDFROMUSERID GET service call Started");
+		response =  (ResponseWrapper<?>) restClientService.getApi(ApiName.GETVIDFROMUSERID, pathSegments, "", "", ResponseWrapper.class);
```

Comment:
Change the api name also to get individual id from user id

Reply:
Renamed  Api name to GETINDIVIDUALIDFROMUSERID